src/structlog/_log_levels.py

```diff
@@ -67,10 +67,6 @@ def add_log_level(
     return event_dict
 
 
-def _nop(*args: Any, **kw: Any) -> Any:
-    return None
```

Comment:
Found no other uses of this method other than the one being changed below, so removed it.

Reply:
I’m on vacation and phone-only, but is there a reason to pull the definition as a closure? That causes both runtime and memory overhead to create the same method over and over again?

Reply:
Hadn't considered a memory impact, I figured the existing logic already made different methods if the min_level was low enough. But, fair point, I'll switch it back so the only change is the signature of `_nop`.